sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.3.1 (Unreleased)
 
 #### Features Added
+- GA release of integrated cache functionality.
```

Comment:
please add a link to this PR here as well the way it's done in the Java SDK, feel free to merge after that :)